### DISCUSSION TITLE
Display SH1107 128x128 IIC support

### DISCUSSION THREAD

**Discussion by driu8808** [arendst/Tasmota#20713] [Q&A]
Hi all,
can anyone help me with the configuration of a display with SH1104 IIC driver?
I didn't find anything online.
Thanks

**Comment by sfromis** [comment]
Not clear if you mean SH1107 or SH1104, but here's a uDisplay config for 64x128 SH1107:
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/SH1107_M5Stick_mono_display.ini

**Reply by driu8808** [reply]
Hi,
I'm talking about the SH1107 (https://it.aliexpress.com/item/1005005985391678.html?spm=a2g0o.order_list.order_list_main.5.21ef36969dEMBe&gatewayAdapt=glo2ita).

The uDisplay config for 64x128 SH1107 referer to the SPI version but I've the IIC version 128x128.
Thanks
